Trying to improve the French translation

SamThill2022 opened the pull request:
Changing a few words so it sounds more natural

SamThill2022 committed:
Create index-fr.2.html

SamThill2022 committed:
Update index-fr.2.html

aatishb commented:
Hi, thanks for submitting this. Could you rename the file to index-fr.html so I can see the specific edits?

SamThill2022 committed:
Add files via upload

SamThill2022 commented:
Hi, thanks for your consideration, I directly uploaded the new file> let me know if it worked? I just started to use Git

nlehuen commented on index-fr.2.html:
```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
```
Can you please delete this file? It's redundant since actual changes are done in index-fr.html.

nlehuen commented on index-fr.html:
```diff
@@ -30,7 +30,7 @@
   <meta property="og:url" content="https://aatishb.com/maskmath/" />
   <meta property="og:image" content="https://aatishb.com/maskmath/assets/twitter-banner.png" />
-  <meta property="og:description" content="Même des masques partiellement efficaces peuvent stopper la diffusion de la COVID-19, à condition qu'assez de gens en portent." />
+  <meta property="og:description" content="Même des masques partiellement efficaces peuvent stopper la diffusion de COVID-19, à condition qu'assez de gens en portent." />
```
I see that you removed "la" before "COVID-19" everywhere. There are [some debates](https://fr.wikipedia.org/wiki/Maladie_%C3%A0_coronavirus_2019#Europe_francophone) on whether COVID-19 should be considered masculine or feminine (i.e. used with "le" or "la"). In general, official wording uses the feminine gender, but popular usage seem to be masculine.

In any case, removing "la" entirely doesn't seem correct to me. We could either have "la diffusion de la COVID-19" or "la diffusion du COVID-19" (since "de le" is conflated into "du" in French). But "la diffusion de COVID-19" doesn't work well, it gives the impression that COVID-19 is some kind of brand or private noun.

nlehuen commented on index-fr.html:
```diff
@@ -42,7 +42,7 @@
       <div>
         <h1>Le pouvoir multiplicateur des masques</h1>
-        <h3>Un essai interactif sur la manière dont les masques peuvent mettre un terme à la COVID-19</h3>
+        <h3>Un essai interactif sur la manière dont les masques peuvent mettre un terme à COVID-19</h3>
```
Same here, it should be either "à la COVID-19" or "au COVID-19" (since "à le" is conflated into "au").

nlehuen replied:
"au COVID-19" ("à COVID-19" is not correct for either masculine or feminine gender of COVID-19).

nlehuen commented on index-fr.html:
```diff
@@ -108,14 +107,14 @@ <h2>Diffusez le message, pas le virus</h2>
     <p>
       Il est important de se souvenir qu'aucun masque n'est parfait.
-      Même les masques FFP2 ou N95 recommandés pour le personnel de santé ne garantissent de bloquer que <a href="https://www.youtube.com/watch?v=eAdanPfQdCA">95% des particules les plus fines</a> (et cela seulement si vous les <a href="https://www.n95decon.org/publications">portez correctement</a>).
+      Même les masques FFP2 ou N95 recommandés pour le personnel de santé ne garantissent pas de bloquer que <a href="https://www.youtube.com/watch?v=eAdanPfQdCA">95% des particules les plus fines</a> (et cela seulement si vous les <a href="https://www.n95decon.org/publications">portez correctement</a>).
```
"pas" is not correct here. We want to say that the mask guarantee a limited level of protection, and "pas" would introduce a double negative. I agree that the sentence is a bit uncomfortable to read though, so I'm open to a full rewrite if you can find something better.

nlehuen commented on index-fr.html:
```diff
@@ -149,7 +148,7 @@ <h2>Pourquoi les masques nous protègent doublement</h2>
   <div class="container">
 
-    <p>Et si à la fois la personne contagieuse <i>et</i> la personne saine portent un masque ?</p> 
+    <p>Et si simultanement  la personne contagieuse <i>et</i> la personne saine portent un masque ?</p>
```
```suggestion
    <p>Et si simultanément la personne contagieuse <i>et</i> la personne saine portent un masque ?</p>
```

nlehuen commented on index-fr.html:
```diff
@@ -164,8 +163,8 @@ <h2>Pourquoi les masques nous protègent doublement</h2>
-      Quand on y pense, c'est surprenant d'un masque efficace à 50% puisse réduire le risque d'infection de 75%.
-      C'est possible parce que lorsque les deux personnes portent des masques, la probabilité d'infection est divisée deux fois.
+      Quand on y pense, c'est surprenant qu'un masque efficace à 50% puisse réduire le risque d'infection de 75%.
+      C'est possible parce que lorsque les deux personnes portent des masques, la probabilité d'infection est divisée par deux.
```
Here we are trying to say that the infection probability was reduced twice (once for the first person, then another time for the other person). It's different from saying that the probability was divided by two.

nlehuen commented on index-fr.html:
```diff
@@ -197,7 +196,7 @@ <h2>Pourquoi les masques nous protègent doublement</h2>
 
       <div class="caption">
-        Les quatre modes de transmission de la COVID-19. Cela suppose un masque efficace à 50%.
+        Les quatre modes de transmission de COVID-19. Cela suppose un masque efficace à 50%.
```
"de la" or "du".

nlehuen commented on index-fr.html:
```diff
@@ -297,7 +296,7 @@ <h2>De la personne à la population</h2>
 
       <div class="caption">
-        Les quatre modes de transmission de la COVID-19 de personne à personne, en supposant que 50% des gens portent un masque efficace à 50%.
+        Les quatre modes de transmission de COVID-19 de personne à personne, en supposant que 50% des gens portent un masque efficace à 50%.
```
"de la" or "du".

nlehuen commented on index-fr.html:
```diff
@@ -497,7 +496,7 @@ <h2>Comment arrêter une épidémie</h2>
 
     <p>
-      Vous pouvez utiliser le premier curseur pour faire varier le R0, qui pour la COVID-19 est entre 2 et 3 (en l'absence de toute mesure de santé publique comme la distanciation sociale, qui réduit le R0).
+      Vous pouvez utiliser le premier curseur pour faire varier le R0, qui pour COVID-19 est entre 2 et 3 (en l'absence de toute mesure de santé publique comme la distanciation sociale, qui réduit le R0).
```
"la" or "le"

nlehuen replied:
```suggestion
      Vous pouvez utiliser le premier curseur pour faire varier le R0, qui pour le COVID-19 est entre 2 et 3 (en l'absence de toute mesure de santé publique comme la distanciation sociale, qui réduit le R0).
```

nlehuen commented on index-fr.html:
```diff
@@ -535,12 +534,12 @@ <h2>Comment arrêter une épidémie</h2>
 
     <p>
-      Pour stopper la diffusion de la COVID-19, nous devons maintenir le R0 en dessous de 1.
+      Pour stopper la diffusion de COVID-19, nous devons maintenir le R0 en dessous de 1.
```
"la" or "le"

nlehuen replied:
Sorry, I mean "de la" or "du".

nlehuen commented on index-fr.html:
```diff
@@ -535,12 +534,12 @@ <h2>Comment arrêter une épidémie</h2>
 
     <p>
-      Donc combien de gens doivent porter un masque efficace à 50% pour stopper la diffusion de la COVID-19 ?
+      Donc combien de gens doivent porter un masque efficace à 50% pour stopper la diffusion de COVID-19 ?
```
"de la" or "du"

nlehuen commented on index-fr.html:
```diff
@@ -623,7 +622,7 @@ <h2>Le coût humain</h2>
       Quand très peu de gens portent un masque, nous nous trouvons en haut de la colline, et la plupart des gens finiront par être infectés.
       Mais chaque pas vers la droite nous fait descendre plus bas, plus vite.
-      Donc même des masques partiellement efficaces, et partiellement utilisés, peuvent aider à réduire la diffusion de la COVID-19.
+      Donc même des masques partiellement efficaces, et partiellement utilisés, peuvent aider à réduire la diffusion de COVID-19.
```
"de la" or "du"

nlehuen commented on index-fr.html:
```diff
@@ -654,16 +653,16 @@ <h2>Le coût humain</h2>
 
     <p>
-      Nous voulons tous descendre la colline et arrêter la diffusion de la COVID-19.
+      Nous voulons tous descendre la colline et arrêter la diffusion de COVID-19.
```
"de la" or "du"

nlehuen commented on index-fr.html:
```diff
@@ -654,16 +653,16 @@ <h2>Le coût humain</h2>
     <p>Ensemble, nous pouvons descendre la colline.</p>
     
-    <p>Ensemble, nous pouvons stopper la COVID-19.</p>
+    <p>Ensemble, nous pouvons stopper COVID-19.</p>
```
"la" or "le".

nlehuen commented on index-fr.html:
```diff
@@ -691,7 +690,7 @@ <h2>Crédits et références</h2>
     <p>
       Voilà <a href="https://static1.squarespace.com/static/5e8126f89327941b9453eeef/t/5f2c4463a5c9f75a38d2b26f/1596736614213/N95DECON_cloth_mask_breathability_filtrationtechnical_report_v1_200804.pdf">quelques</a> <a href="https://www.maskfaq.com/test-results">ressources</a> <a href="http://built-envi.com/what-kind-of-mask-should-i-be-wearing-to-protect-against-covid-19/">utiles</a> sur les capa ...
-      Bien que les masques aient une efficacité variable selon le matériau et l'ajustement au visage, certains matériaux peuvent <a href="https://static1.squarespace.com/static/5e8126f89327941b9453eeef/t/5f2c4463a5c9f75a38d2b26f/1596736614213/N95DECON_cloth_mask_breathability_filtrationtechnical_report_v1_200804.pdf">dépasser</a> <a href="https://www.maskfaq.com/test-results">80%</a> d'efficacité ...
+      Bien que les masques aient une efficacité variable selon le matériel et l'ajustement au visage, certains matériaux peuvent <a href="https://static1.squarespace.com/static/5e8126f89327941b9453eeef/t/5f2c4463a5c9f75a38d2b26f/1596736614213/N95DECON_cloth_mask_breathability_filtrationtechnical_report_v1_200804.pdf">dépasser</a> <a href="https://www.maskfaq.com/test-results">80%</a> d'efficacité ...
```
Here we are talking about the kind of fabric, so "matériau" seems more adequate. This is also consistent with the rest of the sentence: "certains matériaux...".

nlehuen reviewed (changes requested):
Thanks for your contribution! I agree we can do better in some sentences. My biggest comment is about your suggestion to drop the article altogether in front of "COVID-19" - happy to discuss whether to use the masculine or feminine gender, but I think we should keep an article either way.

SamThill2022 committed:
Delete index-fr.2.html

aatishb commented:
Thanks for reviewing @nlehuen! Appreciate it. Could you tag me whenever this is ready to merge?

SamThill2022 commented:
Hi, I do agree with almost everything that Nicholas suggested and started to make a few more changes> After some research it seems that most popular French journals use "du Covid 19" instead of "de la Covid" even if that's what the "Academie Francaise" recommends.

SamThill2022 committed:
Update index-fr.html

this version should take most suggestions in consideration.

nlehuen commented on index-fr.html:
```diff
@@ -8,7 +8,7 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta name="description" content="Un essai interactif sur la manière dont les masques peuvent mettre un terme à la COVID-19">
+  <meta name="description" content="Un essai interactif sur la manière dont les masques peuvent mettre un terme à la diffusion du COVID-19">
```
We can simply have "mettre un terme au COVID-19".

nlehuen commented on index-fr.html:
```diff
@@ -107,12 +107,12 @@ <h2>Diffusez le message, pas le virus</h2>
     <p>
       Il est important de se souvenir qu'aucun masque n'est parfait.
-      Même les masques FFP2 ou N95 recommandés pour le personnel de santé ne garantissent pas de bloquer que <a href="https://www.youtube.com/watch?v=eAdanPfQdCA">95% des particules les plus fines</a> (et cela seulement si vous les <a href="https://www.n95decon.org/publications">portez correctement</a>).
+      Même les masques recommandés pour le personnel de santé, comme le FFP2 ou le N95 garantissent seulement de bloquer 95% des particules les plus fines</a> (et cela uniquement si vous les <a href="https://www.n95decon.org/publications">portez correctement</a>).
```
I propose yet another formulation, maybe a bit easier to read and closer to the original text?

"Même les masques N95 (ou FFP2) recommandés pour le personnel de santé ne garantissent le blocage que de 95% des particules les plus..."

nlehuen commented on index-fr.html:
```diff
@@ -107,12 +107,12 @@ <h2>Diffusez le message, pas le virus</h2>
     <p>
       Les masques ne garantissent pas la sécurité, elles réduisent le risque.
-      Exactement comme un parapluie, qui ne garantit pas que vous resterez sec, mais qui réduit vos chances d'être mouillé.
+      Prenez l'example d'un parapluie, ce dernier ne garantit pas que vous resterez sec, mais il réduit vos chances d'être mouillé.
```
"Prenez l'example d'un parapluie, qui ne garantit pas que vous resterez sec, mais qui réduit vos chances d'être mouillé."

nlehuen commented on index-fr.html:
```diff
@@ -196,7 +196,7 @@ <h2>Pourquoi les masques nous protègent doublement</h2>
 
       <div class="caption">
-        Les quatre modes de transmission de COVID-19. Cela suppose un masque efficace à 50%.
+        Les quatre modes de transmission du virus du COVID-19. Cela suppose un masque efficace à 50%.
```
To be consistent with the rest of the document, let's just write "transmission du COVID-19".

I know that COVID-19 is the disease, and what gets transmitted is the SARS-Cov-2 virus, but at this point a lot of public discussion simply conflates the virus and the disease, and this document is no exception. And I don't think introducing the distinction will significantly improve the document.

nlehuen reviewed (changes requested):
Thanks, this almost LGTM!

SamThill2022 committed:
Update index-fr.html

Hi, I changed the things you mentioned, thank you for letting me contribute to your project it's really interesting to see how it all works.

nlehuen commented on index-fr.html:
```diff
@@ -108,14 +107,14 @@ <h2>Diffusez le message, pas le virus</h2>
     <p>
       Les masques ne garantissent pas la sécurité, elles réduisent le risque.
-      C'est tout à fait comme un parapluie, qui ne garantit pas que vous resterez sec, mais qui réduit vos chances d'être mouillé.
+      Prenez l'example d'un parapluie, ce dernier ne garantit pas que vous resterez sec, mais il réduit vos chances d'être mouillé.
```
```suggestion
      Prenez l'exemple d'un parapluie, ce dernier ne garantit pas que vous resterez sec, mais il réduit vos chances d'être mouillé.
```

nlehuen commented on index-fr.html:
```diff
@@ -149,7 +148,7 @@ <h2>Pourquoi les masques nous protègent doublement</h2>
   <div class="container">
 
-    <p>Et si à la fois la personne contagieuse <i>et</i> la personne saine portent un masque ?</p> 
+    <p>Et si simultanement la personne contagieuse <i>et</i> la personne saine portent un masque ?</p>
```
```suggestion
    <p>Et si la personne contagieuse <i>et</i> la personne saine portent toutes les deux un masque ?</p>
```

nlehuen reviewed (changes requested):
Thanks for your patience! A last round of suggestions and we're good.

aatishb committed:
Update index-fr.html

Co-authored-by: Nicolas Lehuen <nicolas@lehuen.com>

aatishb committed:
Update index-fr.html

Co-authored-by: Nicolas Lehuen <nicolas@lehuen.com>

aatishb committed:
Update index-fr.html

Co-authored-by: Nicolas Lehuen <nicolas@lehuen.com>
